src/Microsoft.AspNet.Razor/TagHelpers/TagHelperAttributeDescriptor.cs

```diff
@@ -116,7 +117,6 @@ public string TypeName
                 }
 
                 _typeName = value;
-                IsStringProperty = string.Equals(_typeName, typeof(string).FullName, StringComparison.Ordinal);
```

Comment:
> We could leave it around for now  

To clarify, as in keep the `IsStingProperty` private setter & set via setting of `TypeName` right?